website/static/js/wikiTreebeard.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
```

Comment:
The name of the JS module should correspond to the constructor being exported, i.e. rename the module to `wikiMenu`.